ls1intum/Artemis
# Adapt About-us-page to also include information about the exam feature

### Checklist
- [x] All described elements are implemented

### Motivation and Context
Artemis has grown and inludes an exam mode as well. This should also be shown in the about-us-page.
Additional links to all features and a guide are linked there as well.

### Description
A second section is now added to the page after the details about all supported exercise types.
Additionally the title of the page as shown by the browser should not be "[object Object]".

### Steps for Testing


1. Log in to Artemis
2. Navigate to the About-us-page linked on the bottom left corner of every ArTEMiS page.
3. Check the new section about the exam mode:
- That all necessary information is present
- That the German AND Englisch texts are concise and correct (you can change the language by clicking on your name on the top right)
4. Check that the title of the page is meaningful in both languages


### Screenshots
![Bildschirmfoto 2021-03-19 um 20 55 13](https://user-images.githubusercontent.com/38322605/111835830-c0ec2a80-88f5-11eb-91a2-a4ce71e81e7b.png)
